src/initializers/growthbook.ts

```diff
@@ -0,0 +1,25 @@
+import requireInjected from '../require-injected';
+import type * as GrowthbookType from 'growthbook';
+
+let gb: GrowthbookType.GrowthBook;
+
+const createGrowthbook = <AppFeatures extends Record<string, any> = Record<string, any>>(config) => {
+  if (!config.growthbook) return;
+  if (!gb) {
+    const {GrowthBook} = requireInjected('growthbook');
+    gb = new GrowthBook<AppFeatures>({
+      apiHost: 'https://cdn.growthbook.io',
+      clientKey: config.growthbook.clientKey
+    });
+  }
+  return gb;
+};
+
+export default createGrowthbook;
+
+export const getGrowthbook = <AppFeatures extends Record<string, any> = Record<string, any>>() => {
```

Comment:
🔧 Per growthbooks documentation you create and destroy an instance every time https://docs.growthbook.io/lib/js.
If you don't there is a risk of mixing features (e.g. an account with a FF enabled might interfere with what happens to one that has the same FF disabled)

Reply:
Ok, I'll address

Reply:
The problem here though, is that we need to give the caller the responsibility to destroy the instance after they use `getGrowthbook`

Reply:
@nikostoulas any idea if we could somehow handle this?

Reply:
See how we handle this in job-board https://github.com/Workable/job-board/blob/master/src/app/middlewares/growth-book.ts#L22

Reply:
This is the easy part. What if someone calls `getGrowthbook` from a queue handler?

Reply:
You should not expose getGrowthbook. In this case it is the callers responsibility to call destroy.

Reply:
I need to expose, in order to be able to use it on queues, consumers, workers etc

Reply:
I mean you should expose a createGrowthbook as growthbook instances should not be shared!

Reply:
addressed